src/APL/vercel-apl.ts

```diff
@@ -65,23 +65,31 @@ export class VercelAPL implements APL {
   }
 
   private async saveDataToVercel(authData?: AuthData) {
-    debug(`saveDataToVercel with: ${authData}`);
+    debug("saveDataToVercel with: %j", authData);
```

Comment:
Lets write debug messages that are explicit. If we track function called, it can be
`saveDataToVercel() called with "%j` - so its clear that debug says about function invocation. Its also most likely by clear that its beginning of the function

Reply:
Also shouldn't you obfuscate authdata? Maybe some substring(0,4) will work for token?